storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1171,7 +1166,7 @@ static TYPELIB read_free_rpl_typelib = {array_elements(read_free_rpl_names) - 1,
  * myrocks::rocksdb_file_checksum_type */
 static const char *file_checksums_names[] = {
     "CHECKSUMS_OFF", "CHECKSUMS_WRITE_ONLY", "CHECKSUMS_WRITE_AND_VERIFY",
-    NullS};
+    "CHECKSUMS_WRITE_AND_VERIFY_ON_CLONE", NullS};
```

Comment:
Are there tests that need re-recording? `main.mysqld--help-notwin`, maybe others?

Reply:
Doesn't seem so. The related MTR test passes. But I add one more MTR test for this change. It's included in the latest iteration.